mantidproject/mantid
# Fix sphinx warnings from RHEL 7 clean build

Fix the warnings found on the RHEL clean master build

No issue to close on this, PR only.
## Release Notes

Not worth mentioning
## To test

check the build output for warnings related to sphinx docs - Text Search for Warning
